_posts/2015-04-03-april-newsletter.md

```diff
@@ -0,0 +1,250 @@
+---
+layout: newsletter
+title: "Newsletter #5 - TODO"
+category: newsletter
+permalink: /news/2015/april/
+---
+
+Welcome to the fifth newsletter for Neovim.
+
+## General News
+
+### Blackduck 2014 Open Source Rookie
+
+[Blackduck][blackduck-site], a company that helps other corporations in the
+adoption of open source software, chose Neovim as one of the projects to feature
+of this last year. The company annually recognizes some of the top new open
+source projects that started during the year.
+
+### ICCF Holland Donations
+
+Mentioned in the last letter, [Unixstickers][iccf-unixstickers] has added
+[Neovim stickers][iccf-stickers] to their inventory. Every sticker that is
+bought increases the amount donated to [ICCF Holland][iccf-site], Bram
+Moolenaar's charity.
+
+Since they've been added, $106 has been raised which Unixstickers [just
```

Comment:
I believe I am? Unixstickers site defaults to USD (top right) so I just assumed that the amount on the side of each [sticker page](http://www.unixstickers.com/stickers/software_stickers/neovim-text-editor-logo-shaped-sticker) was USD?